css/style.css

```diff
@@ -411,112 +358,156 @@ content: "+";
-  font-style: normal;
-  line-height: 26px;
-  color: #fff;
+.footer-text {
   font-size: 16px;
+  line-height: 1.6;
 }
 
-.joinus-text{
-  font-family: "Poppins", "Roboto", sans-serif;
-  font-weight: 400;
-  font-style: normal;
+.joinus-text {
   font-size: 16px;
-  line-height: 26px;
-  color: #fff;
+  line-height: 1.6;
 }
 
-.logo-footer-text{
-  font-family: "Poppins", "Roboto", sans-serif;
-  font-weight: 400;
-  font-style: normal;
+.logo-footer-text {
   font-size: 30px;
-  line-height: 42px;
+  line-height: 1.4;
 }
 
-.other-nav, .documentation-heading{
+.other-nav, .documentation-heading {
   font-size: 16px;
-  line-height: 24px;
+  line-height: 1.5;
 }
 
-.other-nav-list, .documentation-list, .footer-copy{
-  font-family: "Poppins", "Roboto", sans-serif;
-  font-weight: 400;
-  font-style: normal;
+.footer-copy {
   font-size: 14px;
-  line-height: 22px;
+  line-height: 1.5;
+}
+.documentation-list {
+  font-size: 14px;
+  line-height: 1.5;
 }
 
-.other-nav-link, .documentation-link{
+.other-nav-list {
+  font-size: 14px;
+  line-height: 1.5;
+}
+
+.other-nav-link, .documentation-link {
+  font-family: "Poppins","Roboto", sans-serif;
+  font-size: 14px;
+  line-height: 1.5;
+  color: C2C3C5;
   text-transform: capitalize;
 }
+.subscription-form-heading{
+  font-family: "Bakbak One", "Roboto", sans-serif;
+  font-size: 16px;
+  line-height: 1.5;
+}
 
-.button-subscribe{
+.button-subscribe
+ {
   font-family: "Bakbak One", "Roboto", sans-serif;
-	font-weight: 400;
-	font-style: normal;
   color: #040B11; 
   font-size: 16px;  
-  line-height: 24px;
+  line-height: 1.5;
 }
 
-.product-tittle{
+.product-tittle {
+  font-family: "Bakbak One", "Roboto", sans-serif;
   font-size: 44px;
-  line-height: 56px;
+  line-height: 1.2;
 }
 
-.product-text{
-  font-family: "Poppins","Roboto", sans-serif;
-  font-weight: 400;
-  font-style: normal;
-  font-size: 20px;
-  line-height: 30px;
+.product-description{
+  font-family: "Bakbak One", "Roboto", sans-serif;
+  font-size: 24px;
+  line-height: 1.2;
+}
+
+.product-text {
   color: #C2C3C5;
 }
 
-.product-termin{
-  font-family: "Poppins","Roboto", sans-serif;
-  font-weight: 400;
-  font-style: normal;
+.product-termin {
   font-size: 16px;
-  line-height: 26px;
+  line-height: 1.6;
   color: #C2C3C5;
 }
 
-.product-price{
+.product-price {
+  font-family: "Bakbak One", "Roboto", sans-serif;
   font-size: 36px;
-  line-height: 44px;
+  line-height: 1.2;
   text-transform: uppercase;
 }
 
-.price{
+.price {
   color:#21E786;
 }
 
-.button-buy{
+.button-buy {
   font-family: "Bakbak One", "Roboto", sans-serif;
-	font-weight: 400;
-	font-style: normal;
   color: #040B11; 
   font-size: 16px;  
-  line-height: 24px;
+  line-height: 1.5;
+}
+
+
+
+
+/* ---------styles for images--------- */
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+body {
+  background-image: url("../img/star.svg"), url("../img/star.svg"),url("../img/star.svg"),url("../img/star.svg"),url("../img/star.svg"),;
```

Comment:
remove the comma at the end of the property, because of this the styles are not applied